src/collection.ts

```diff
@@ -86,7 +86,6 @@ import {
   Callback,
   checkCollectionName,
   DEFAULT_PK_FACTORY,
-  emitWarningOnce,
```

Comment:
This has nothing to do with this ticket specifically, but was failing the lint task on CI.